README.md

```diff
@@ -82,8 +82,9 @@ For contribute in this project, you need to follow the next steps:
 2. Clone your forked repository
 3. Create a new branch
 4. Make your changes
-5. Push your changes to your forked repository
-6. Make a pull request
+5. Run the tests
```

Comment:
Thanksss

Reply:
but why would you  run the test after the pr?

Reply:
![image](https://github.com/esgaelramos/YourMoneyTrackerMail/assets/89751740/fc95dbc5-fff1-4511-b14c-41e422223bf4)